apps/cowswap-frontend/src/modules/orderProgressBar/pure/LottieContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode, useRef, useEffect } from 'react'
+
+import Lottie from 'lottie-react'
+import styled from 'styled-components/macro'
+
+const LottieWrapper = styled.div`
+  --size: 100%;
+  width: var(--size);
+  height: var(--size);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+
+  /* Ensure Lottie container uses full space */
+  > div {
+    width: var(--size) !important;
+    height: var(--size) !important;
+  }
+
+  svg {
+    width: var(--size) !important;
+    height: var(--size) !important;
+  }
+`
+
+interface FullSizeLottieProps {
+  animationData: unknown
+  loop?: boolean
+  autoplay?: boolean
+}
+
+export function FullSizeLottie({ animationData, loop = true, autoplay = true }: FullSizeLottieProps): ReactNode {
+  const lottieRef = useRef<HTMLDivElement>(null)
+
+  useEffect((): (() => void) | void => {
```

Comment:
It's not necessary to define types for `useEffect` callbacks, no?

Reply:
Correct, fixed now.